srcpkgs/wayfire/template

```diff
@@ -1,10 +1,11 @@
 # Template file for 'wayfire'
 pkgname=wayfire
-version=0.7.0
-revision=3
+version=0.7.1
+revision=1
 _utils_commit=f45641beef46babdc8f1b8d18a924e72beaf8ee6
 _touch_commit=b1075c54a280f913edc26b9757262f4f9d6b62b0
 build_style=meson
+configure_args="-Dprint_trace=false"
```

Comment:
I think the patch to add this could be expanded to completely remove the `execinfo` dependency... Not for now, just a note.